docs/spire.md

```diff
@@ -6,13 +6,58 @@ weight: 1660
 -->
 ⚠️ This is a work in progress: SPIRE support is not yet functional
 
-TaskRun result attestations is currently an alpha experimental feature. Currently all that is implemented is support for configuring Tekton to connect to SPIRE. See TEP-0089 for details on the overall design and feature set.
+TaskRun result attestations is currently an alpha experimental feature. Currently all that is implemented is support for configuring Tekton to connect to SPIRE and enabling TaskRun to sign and verify the TaskRunStatus. See TEP-0089 for details on the overall design and feature set.
 
 This being a large feature, this will be implemented in the following phases. This document will be updated as we implement new phases.
 1.  Add a client for SPIRE (done).
 2.  Add a configMap which initializes SPIRE (in progress).
-3.  Modify TaskRun to sign and verify TaskRun Results using SPIRE.
-4.  Modify Tekton Chains to verify the TaskRun Results.
+3.  Modify TaskRun to sign and verify TaskRunStatus using SPIRE.
+4.  Modify Tekton Chains to verify the TaskRunStatus.
+
+When the TaskRun result attestations feature is enabled, all TaskRuns will produce a signature alongside its results, which can then be used to validate its provenance. For example, a TaskRun result that creates user-specified results `commit` and `url` would look like the following. `SVID`, `RESULT_MANIFEST`, `RESULT_MANIFEST.sig`, `commit.sig` and `url.sig` are generated attestations by the integration of SPIRE and Tekton Controller.
```

Comment:
> TaskRun result attestations feature is enabled

which feature is this referring to? how can it be enabled? please be specific, and add a link to the feature

Reply:
Added a link on "enabled" to point to how to enable this feature.